code/modules/organs/internal/_internal.dm

```diff
@@ -63,16 +63,34 @@
 		. += "[get_wound_severity(get_scarring_level())] scarring"
 
 /obj/item/organ/internal/is_usable()
-	return ..() && !is_broken()
+	if(robotize_type)
+		var/datum/robolimb/R = all_robolimbs[robotize_type]
+		if(!R.malfunctioning_check(owner))
+			return TRUE
+	else
+		return ..() && !is_broken()
 
 /obj/item/organ/internal/proc/is_damaged()
 	return damage > 0
 
-/obj/item/organ/internal/robotize()
+/obj/item/organ/internal/robotize(var/company)
 	..()
 	min_bruised_damage += 5
 	min_broken_damage += 10
 
+	if(!company)
+		company = "Unbranded"
```

Comment:
wouldn't it be easier to set it as `/obj/item/organ/internal/robotize(var/company = "Unbranded")`?